sljit_src/sljitNativeRISCV_common.c

```diff
@@ -88,6 +89,10 @@ static const sljit_u8 freg_map[SLJIT_NUMBER_OF_FLOAT_REGISTERS + 3] = {
 #define BGE		(F3(0x5) | OPC(0x63))
 #define BLTU		(F3(0x6) | OPC(0x63))
 #define BGEU		(F3(0x7) | OPC(0x63))
+#if defined __riscv_zbb
+#define CLZ		(F7(0x30) | F3(0x1) | OPC(0x13))
+#define CTZ		(IMM_I(0x601) | F3(0x1) | OPC(0x13))
```

Comment:
IMM is not used for opcode definitions. Combine from F7 and `(((sljit_ins)0x...) << ...)`